### PR TITLE
[what] update cmake compilation configuration in windows

### PR DESCRIPTION
**Pull request by HR1025**
[why] support some other IDE like VScode or clion

[how]
1. update CMakeLists.txt according to *.vcxproj
2. include some header like `windows.h` to support raw cmake compilation
3. exclude libcmt.lib

### CODE REVIEW

File: Windows/GEDebugger/GEDebugger.cpp [hunk 1]
```diff
@@ -22,6 +22,7 @@
 #include <vector>
 
 #include "Common/CommonWindows.h"
+#include <windows.h>
```
Comment (unknownbrackets) [inline Windows/GEDebugger/GEDebugger.cpp:25]: Please use `#include "Common/CommonWindows.h"` instead which defines WIN32_LEAN_AND_MEAN.  Why did you need to add this twice?

-[Unknown]

File: Windows/W32Util/DarkMode.h [hunk 2]
```diff
@@ -3,7 +3,8 @@
 #define WIN32_LEAN_AND_MEAN
 #define NOMINMAX
 #include <Windows.h>
-#include <CommCtrl.h>
+#include <windows.h>
```
Comment (unknownbrackets) [inline Windows/W32Util/DarkMode.h:6]: Why do we have `Windows.h` and `windows.h`?  Although `"Common/CommonWindows.h` is preferred.

-[Unknown]

**Comment by HR1025** [comment]
Maybe this is redundant, I'll check again.

I used global substitution, from:
```cpp
#include <commctrl.h>
```
to
```cpp
#include <windows.h>
#include <commctrl.h>
```

Because I found that some places seemed to be missing the header file when I was compiling.

I'll double check and make some modifications

**Review by hrydgard — APPROVED** [review]

**Review by unknownbrackets — APPROVED** [review]

**Comment by unknownbrackets** [comment]
Thanks for making the updates, and cleaning up some cases that weren't consistent.

-[Unknown]
